src/renderer/store/modules/utils.js

```diff
@@ -523,7 +523,12 @@ const actions = {
 
     if (payload.watchProgress > 0 && payload.watchProgress < payload.videoLength - 10) {
       if (typeof cmdArgs.startOffset === 'string') {
-        args.push(`${cmdArgs.startOffset}${payload.watchProgress}`)
+        if (cmdArgs.startOffset.endsWith('=')) {
+          args.push(`${cmdArgs.startOffset}${payload.watchProgress}`)
+        } else {
+          // special handling for SMPlayer and similar
+          args.push(cmdArgs.startOffset, Math.trunc(payload.watchProgress))
+        }
```

Comment:
Just better comments
```suggestion
        if (cmdArgs.startOffset.endsWith('=')) {
          // For players using `=` in arguments
          // e.g. vlc --start-time=xxxxx
          args.push(`${cmdArgs.startOffset}${payload.watchProgress}`)
        } else {
          // For players using space in arguments
          // e.g. smplayer -start xxxxx
          args.push(cmdArgs.startOffset, Math.trunc(payload.watchProgress))
        }
```